src/frontend/demo/src/views/roomSettings.tsx

```diff
@@ -0,0 +1,24 @@
+import { LayoutWithSidebar, RoomConfig, useTranslations } from '@jitsi-magnify/core';
+import { defineMessages } from 'react-intl';
+import { Navigate, useParams } from 'react-router-dom';
+
+export const messages = defineMessages({
+  roomSettingsTitle: {
+    id: 'app.roomSettingsTitle',
+    description: 'H1 page title for the room settings view',
+    defaultMessage: 'Room Settings',
+  },
+});
+
+export default function RoomSettingsView() {
+  const intl = useTranslations();
+  const { roomName } = useParams();
+
+  if (!roomName) return <Navigate to="/room-not-found" replace />;
+
+  return (
+    <LayoutWithSidebar title={intl.formatMessage(messages.roomSettingsTitle)}>
+      <RoomConfig roomName={roomName || 'def'} />
```

Comment:
With the if at line 17, no default value could be used.
```suggestion
      <RoomConfig roomName={roomName} />
```